caikit/runtime/service_factory.py

```diff
@@ -266,3 +267,38 @@ def _get_and_filter_modules(
             excluded_modules,
         )
         return clean_modules
+
+
+def get_request(
+    module_class: Type[ModuleBase],
+    input_streaming: bool = False,
+    output_streaming: bool = False,
+) -> Type[DataBase]:
+    """Helper function to return the request DataModel for the Module Class"""
+    if input_streaming and output_streaming:
+        request_class_name = f"BidiStreaming{module_class.TASK_CLASS.__name__}Request"
```

Comment:
@prashantgupta24 since this only requires the task, what do you think about supplying the task as the parameter instead of the module?

I'm thinking ahead a little bit here: we've had requests for multi-modal modules that support multiple tasks so we might also have a future where there are multiple inference requests for a single module